src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        int numberOfBall = new Random().nextInt(100);
+        Ball ball = new Ball();
+        ball.SetColor(colorSupplier.toString());
+        ball.SetNumber(numberOfBall);
```

Comment:
The method name `SetNumber` does not follow Java naming conventions. It should be `setNumber`.